core/trino-main/src/main/java/io/trino/sql/planner/SplitSourceFactory.java

```diff
@@ -154,14 +156,15 @@ public Map<PlanNodeId, SplitSource> visitExplainAnalyze(ExplainAnalyzeNode node,
         @Override
         public Map<PlanNodeId, SplitSource> visitTableScan(TableScanNode node, Void context)
         {
-            return visitScanAndFilter(node, Optional.empty());
+            SplitSource splitSource = createSplitSource(node.getTable(), node.getAssignments(), Optional.empty());
+
+            splitSources.add(splitSource);
+
+            return ImmutableMap.of(node.getId(), splitSource);
         }
 
-        private Map<PlanNodeId, SplitSource> visitScanAndFilter(TableScanNode node, Optional<FilterNode> filter)
+        private SplitSource createSplitSource(TableHandle table, Map<Symbol, ColumnHandle> columnHandles, Optional<Expression> filterPredicate)
```

Comment:
columnHandles -> assignments

(eg can `columnHandles.values()` have duplicates? what's the meaning of map keys?)

Reply:
renamed to `assignments`